antarest/storage/repository/filesystem/bucket_node.py

```diff
@@ -45,5 +45,7 @@ def build(self, config: StudyConfig) -> TREE:
         os.chdir(current_dir)
         return children
 
-    def validate(self, data: JSON) -> None:
-        pass  # no validation for bucket node
+    def validate(
```

Comment:
even though, one can simply check if the return message array length is 0 to check the validity. It would be better to return a tuple (bool, List[str]) with validation boolean result and possibly just warning non blocking messages

Reply:
also we could add an other optional input "raise_on_error" (default False) that would raise at first exception (if possible), in order not to bother parsing the whole broken simulation if needed.
that could be used to quickly invalidate studies before launch.

Reply:
with python a condition return false if a list is empty. Therefore we can rename method to check_errors to have some line like ) `if study.check_error()`. It will avoid to have a boolean